apps/ml/site/analysis/analysis.jag

```diff
@@ -403,8 +403,8 @@
                     handleNotification(successText, '#notification-area', 'success');
                 },
                 error : function(res){
-                var errorText = res.responseText
-                    handleNotification(errorText, '#notification-area', 'warning');
+                var errorText = JSON.parse(res.responseText)["exception"];
```

Comment:
Use a common way to access JSON elements. Either res["responseText"]["exception"] or res.responseText.exception